webrtc-stats.html

```diff
@@ -1064,6 +1075,99 @@
           </section>
         </div>
       </section>
+      <section id="contributingsourcestats-dict*">
+        <h3>
+          <dfn>RTCRTPContributingSourceStats</dfn> dictionary
+        </h3>
+        <p>
+          The <code>RTCRTPContributingSourceStats</code> dictionary represents
+          the measurement metrics for a contributing source (CSRC) that is
+          contributing to an incoming RTP stream. Each contributing source
+          produces a stream of RTP packets, which are combined by a mixer into
+          a single stream of RTP packets that is ultimately received by the
+          WebRTC endpoint. Information about the sources that contributed to
+          this combined stream may be provided in the CSRC list or [[RFC6465]]
+          header extension of received RTP packets.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCRTPContributingSourceStats : RTCStats {
+             unsigned long contributorSsrc;
+             DOMString     rtpStreamStatsId;
+             unsigned long packetsContributedTo;
+             double        audioLevel;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCRTPContributingSourceStats</a> Members
+            </h2>
+            <dl data-link-for="RTCRTPContributingSourceStats"
+                data-dfn-for="RTCRTPContributingSourceStats"
+                class="dictionary-members">
+              <dt>
+                <dfn><code>contributorSsrc</code></dfn> of type <span class=
+                 "idlMemberType"><a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  The SSRC identifier of the contributing source represented by
+                  this stats object, as defined by [[!RFC3550]]. It is a 32-bit
+                  unsigned integer that appears in the CSRC list of any packets
+                  the relevant source contributed to.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>rtpStreamStatsId</code></dfn> of type <span class=
```

Comment:
Most Id names leave out "Stats", the only exception being rtcpTransportStatsId which arguably should be rtcpTransportId (transportId used in several places) for consistency and associateStatsId whose name is vague about what type it is referencing.

How about inboundRtpStreamId?